interfaces.go

```diff
@@ -190,6 +204,16 @@ type Logger interface {
 	Clone() Logger
 }
 
+type EventBuilder interface {
```

Comment:
Lets add godoc comment what is this, maybe with mention what type this interface covers like here https://github.com/moira-alert/moira/blob/master/senders/mattermost/client.go#L5

Reply:
Maybe move interfaces `EventBuilder` and `Logger` to logging pkg. Or move at least `EventBuilder` . So it will be more near to usage. I don't like interfaces.go in root of project, it is trash I consider. Is it fine, how do you think?

Reply:
Added godoc.

Reply:
Created task